tests/src/test/java/alluxio/shell/command/CopyFromLocalCommandTest.java

```diff
@@ -358,6 +362,21 @@ public void copyFromLocal() throws IOException, AlluxioException {
   }
 
   @Test
+  public void copyFromLocalMustCacheThenCacheThrough() throws Exception {
+    File file = mTestFolder.newFile();
+    try (Closeable c = new ConfigurationRule(PropertyKey.USER_FILE_WRITE_TYPE_DEFAULT,
+        WriteType.MUST_CACHE.toString()).toResource()) {
+      Assert.assertEquals(0, mFsShell.run("copyFromLocal", file.getAbsolutePath(), "/"));
+    }
+    try (Closeable c = new ConfigurationRule(PropertyKey.USER_FILE_WRITE_TYPE_DEFAULT,
+        WriteType.CACHE_THROUGH.toString()).toResource()) {
+      mOutput.reset();
+      mFsShell.run("copyFromLocal", file.getAbsolutePath(), "/");
+    }
+    Assert.assertThat(mOutput.toString(), CoreMatchers.containsString("already exists"));
```

Comment:
If we make this match an exact string, the test will break any time the implementation changes its wording. As long as the reported error contains "already exists" I'm confident that it will help the user figure out why their `copyFromLocal` is failing.